drivers/ieee802154/Kconfig

```diff
@@ -57,6 +57,27 @@ config IEEE802154_VENDOR_OUI
 
 endif # IEEE802154_VENDOR_OUI_ENABLE
 
+config IEEE802154_UICR_EUI64_ENABLE
+	bool "Enables using EUI64 value stored in UICR registers"
+	depends on !IEEE802154_VENDOR_OUI_ENABLE
+	help
+	  This option enables setting custom vendor EUI64 value
+	  stored in User information configuration registers (UICR).
+	  Notice that this disables the default setting of EUI64
+	  value from Factory information configuration registers
+	  (FICR).
+
+if IEEE802154_UICR_EUI64_ENABLE
+
+config IEEE802154_UICR_EUI64_REG
+	int "UICR base register for the EUI64 value"
```

Comment:
lets add range here not to exceed the registers count

Reply:
Sure.